__tests__/Utils/updateQueryStringToUrl.test.ts

```diff
@@ -0,0 +1,33 @@
+import { render, fireEvent, screen } from '@testing-library/react';
+import { updateQueryStringToUrl } from '@/utils/updateQueryStringToUrl';
+import { NextRouter } from 'next/router';
+
+// Mock Next.js router
+const mockRouter: NextRouter = {
+    push: jest.fn() as any,
```

Comment:
Can you remove `as any` as it isn't a good practice to follow.

Reply:
Done.